src/traits/has_members.rs

```diff
@@ -0,0 +1,24 @@
+use crate::act::node::{candid::CandidType, declaration::Declare, Declaration};
+
+pub trait HasMembers {
+    fn get_members(&self) -> Vec<CandidType>;
+
```

Comment:
Add a get_inline_prefix(parental_prefix: String) and use that instead of the name whatever

Reply:
Do that for has enclose type and any other trait as well

Reply:
Add polish to all of this

Reply:
Look to see if we can make a parent trait that all of these traits can extend.